### PR TITLE
add(main): new package robin-map

### PR DESCRIPTION
**Pull request by T-Dynamos**
#11674

### CODE REVIEW

**Comment by sylirre** [comment]
Instead of adding one-by-one each of Blender dependency, please add them in one pull request. Just to note that if Blender packaging will be failed, the standalone library packages will be mostly useless.

File: packages/robin-map/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Tessil/robin-map/
+TERMUX_PKG_DESCRIPTION="C++ implementation of a fast hash map and hash set using robin hood hashing "
```
Comment (Grimler91) [inline packages/robin-map/build.sh:2]: Please remove extra space at end of description

**Review by Grimler91 — CHANGES_REQUESTED** [review]

**Comment by T-Dynamos** [comment]
Please merge this

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**